### DISCUSSION TITLE
Perlin noise extension problem

### DISCUSSION THREAD

**Discussion by Flappy25** [BlueDome77/Turbowarp-Extension-List#2] [General]
My perlin noise extension isn't true randomness because it uses the Math.random() function

**Comment by BlueDome77** [comment]
> My perlin noise extension isn't true randomness because it uses the Math.random() function

What's the problem then?

**Reply by Flappy25** [reply]
It's not real perlin noise

**Reply by BlueDome77** [reply]
So should I take it off of the list then or just rename it?
